crates/vhost-device-gpio/src/backend.rs

```diff
@@ -161,8 +173,37 @@ impl TryFrom<GpioArgs> for GpioConfiguration {
     }
 }
 
-fn start_backend<D: 'static + GpioDevice + Send + Sync>(args: GpioArgs) -> Result<()> {
-    let config = GpioConfiguration::try_from(args).unwrap();
+fn start_device_backend<D: GpioDevice>(device: D, socket: String) {
```

Comment:
This drops the `-> Result` somehow... Did this maybe get messed up when you rebased on my refactoring of this? You also revert a couple of `.map_err()?` to `.unwrap()` below.

Reply:
Ah ok. It is just the history that is messed up... The rebasing weirdness gets fixed in the last commit again. It was just confusing to review....

Reply:
yeah start_device_backend is different - and does gain a result in the final version.